src/test/java/io/cdap/plugin/gcp/bigquery/action/BigQueryArgumentSetterConfigTest.java

```diff
@@ -0,0 +1,46 @@
+package io.cdap.plugin.gcp.bigquery.action;
```

Comment:
missing copyright. 
Are you able to build the plugins using - `mvn clean package -DskipTests`?